### PR TITLE
[OBSDEF-15071] Fix table overflow

### PR DESCRIPTION
**Pull request by Bhatii**
## Summary
This PR fixes:
- Overflow issue in table data

### CODE REVIEW

File: src/datagrid/DataGrid.tsx [hunk 1]
```diff
@@ -1071,7 +1071,7 @@ export class DataGrid extends React.PureComponent<DataGridProps, DataGridState>
     private buildDataGridBody(): React.ReactElement {
         const {allRows} = this.state;
         return (
-            <div className={ClassNames.DATAGRID} style={{overflow: "initial"}}>
+            <div className={ClassNames.DATAGRID}>
```
Comment (akate1) [inline src/datagrid/DataGrid.tsx:1074]: Datagrid Footer is broken due to some changes. Please don't add custom inline styles to fix it we need to fix original issue.

**Review by akate1 — CHANGES_REQUESTED** [review]
Defect 15071 is not related to this PR

File: src/datagrid/DataGrid.tsx (hunk 1 above)
Comment (akate1) [reply src/datagrid/DataGrid.tsx:1074]: @Bhatii along with this please also remove inline style for Daragrid header that u have added in PR https://github.com/EMCECS/clarity-react/pull/244

**Review by akate1 — COMMENTED** [review]
@Bhatii As we are reverting some changes from PR https://github.com/EMCECS/clarity-react/pull/244 to fix this issue. Please re-validate datagrid filter things

**Comment by Bhatii** [comment]
Closed due to related to other bug.
New PR: https://github.com/EMCECS/clarity-react/pull/247

**Pull request closed without merging**